test/integration/test-fixtures/image-test-java-purls/Dockerfile

```diff
@@ -0,0 +1 @@
+FROM anchore/test_images:java-56d52bc
```

Comment:
This is a bit of a hacky way to bring the test fixture into syft. Let me know if it's better to do something else, but I like that this pins the artifact.

Reply:
this is the right approach, but should we also use a digest too?